ngx-tools/src/utilities/changes-util.ts

```diff
@@ -0,0 +1,31 @@
+import { SimpleChanges } from '@angular/core';
+import { objectDeepParse } from './object-deep-parse';
+
+export class ChangesUtil {
```

Comment:
Can we make this name more explicit? Without the context of this PR I'm not sure I'd know what this class is for.

Let's add some JSDoc comments in this class also.

Reply:
Any suggestion for the name? `NgChangeObjectValueParser`?